src/types.ts

```diff
@@ -157,13 +160,29 @@ export interface Service {
 	user?: string;
 	userns_mode?: string;
 
-	volumes?: ListOfUniqueItems<string>;
+	volumes?: ListOfUniqueItems<string | VolumeRef>;
```

Comment:
```suggestion
	volumes?: ListOfUniqueItems<string>;
```

Right?

Reply:
:+1: forgot to remove that after your earlier comments